tc/core/compiler.h

```diff
@@ -19,6 +19,8 @@
 #include <string>
 #include <vector>
 
+#include "tc/core/exceptions.h"
+#include "tc/core/halide_utils.h"
```

Comment:
not worth it, just pass a TreeRef, or better yet do not expose the function and increase the compiler API